server/lib/service/firestore.dart

```diff
@@ -93,6 +140,24 @@ class FirestoreClient {
     return document;
   }
 
+  /// Gets a list of [Document]s in [collection] from Firestore.
+  ///
+  /// The [pageSize] is the maximum number of documents to return. The
+  /// [pageToken] is the token used to retrieve the next page.
+  ///
+  /// Throws an [ApiRequestError] if Firestore API returns an error.
+  Future<ListDocumentsResponse> _listDocuments(String parent, String collection,
+      {int pageSize, String pageToken}) async {
+    final documents = await _api.projects.databases.documents.list(
+      'projects/$_projectId/databases/$_databaseId/documents$parent',
+      collection,
+      pageSize: pageSize,
```

Comment:
How is  the `pageSize` gonna be used in UI? If we do need the `pageSize`? Can we rename it to `maxPageSize`?

Reply:
Currently I don't plan to use pageSize in the UI, but pagination could be possible in the future. `maxPageSize` is a better name though, I'll change it.